Cabal-syntax/src/Distribution/Types/PackageDescription.hs

```diff
@@ -146,7 +146,7 @@ data PackageDescription
         extraTmpFiles  :: [FilePath],
         extraDocFiles  :: [FilePath]
     }
-    deriving (Generic, Show, Read, Eq, Typeable, Data)
+    deriving (Generic, Show, Read, Eq, Ord, Typeable, Data)
```

Comment:
I had to add a number of `Ord` instances after structuring `check` errors (before: sorting was done on output `String`s). It seems to me the new `deriving (Ord)` will not cause problems (most of the types are sum types).